aws-distro-opentelemetry-node-autoinstrumentation/test/aws-metric-attribute-generator.test.ts

```diff
@@ -1167,6 +1200,10 @@ describe('AwsMetricAttributeGeneratorTest', () => {
     testAwsSdkServiceNormalization('Kinesis', 'AWS::Kinesis');
     testAwsSdkServiceNormalization('S3', 'AWS::S3');
     testAwsSdkServiceNormalization('SQS', 'AWS::SQS');
+    testAwsSdkServiceNormalization('SNS', 'AWS::SNS');
+    testAwsSdkServiceNormalization('Lambda', 'AWS::Lambda');
+    testAwsSdkServiceNormalization('SecretsManager', 'AWS::SecretsManager');
+    testAwsSdkServiceNormalization('StepFunctions', 'AWS::StepFunctions');
```

Comment:
Can you verify that we should actually be testing that `SFN` (instead of `StepFunctions`) is normalized to `AWS::StepFunctions`? I see we commonly reference `SFN` in this PR. I suppose in that case, you need to update `normalizeRemoteServiceName` to handle `SFN` for this test to work.